nss-day-cohort-60/national-parks-django-api
# event view

Implemented events view.

**Steps to test**
1. `git fetch origin hp-event-view`
2. `git checkout hp-event-view`
3. Start/restart server
4. Open postman (there is currently no client-side use for event views)
5. Send a GET request to http://localhost:8000/events
6. Send a GET request to http://localhost:8000/events/3
7. Send a POST request to http://localhost:8000/events
8. Here's something you can send. Feel free to change any of it:
 `{
        "name": "dkjngkjdgtnhkj tfkjngrltkjgnlkjdnlk",
        "start_date": "2023-01-28 10:00:00",
        "description": "fdkjngikjntghkjgnkjgfn.",
        "park": 4
}`
9. You should see it serialized in the response. 
10. Send a PUT request to http://localhost:8000/events/10 (should be the event you just created). You can edit all the fields on this, so go ahead and do that. You should see an empty response and a 204 response code. Send a GET request to that same url and see if your changes saved.
11. Send a DELETE request to http://localhost:8000/events/10
12. You should see an empty response again. Send a GET request to http://localhost:8000/events and make sure your event is gone.
